DoSomething/quasar
# Updating newsletter tables to be aware of email unsubs

#### What's this PR do?
This PR joins the user activity table to the derived user_newsletter_subscriptions table so that when a user unsubscribes from email altogether it's recorded as unsubscribing from topics. Pulling the data into the derived table should limit some of the issues we might have with exposing too much to the end user in Looker.
#### Where should the reviewer start?
I've built the table in production via DBT and it can be viewed at `ds_dbt.user_newsletter_subscriptions`
#### How should this be manually tested?
I've run a few queries to verify that this populates the `topic_unsubscribed_at` with the `user_activity.email_unsubscribed_at` date when appropriate and `topic_unsubscribed_at` when the user is still subscribed to email is working properly/unaffected by this change.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
